Author: Alastair McKinstry <mckinstry@debian.org>
Description: Find certain libraries in the /usr/lib/* multiarch directory on Debian.
Last-Updated: 2011-09-14
Forwarded: no

Index: visit-2.3.2/src/CMake/FindVisItMesa.cmake
===================================================================
--- visit-2.3.2.orig/src/CMake/FindVisItMesa.cmake	2011-09-14 09:38:19.000000000 +0100
+++ visit-2.3.2/src/CMake/FindVisItMesa.cmake	2011-09-14 09:38:34.000000000 +0100
@@ -57,7 +57,7 @@
 IF (WIN32)
     SET_UP_THIRD_PARTY(MESA lib/${VISIT_MSVC_VERSION} include MesaGL32 osmesa32)
 ELSEIF(DEBIAN)
-    SET_UP_THIRD_PARTY(MESA lib include GL OSMesa)
+    SET_UP_THIRD_PARTY(MESA lib/${DEB_HOST_MULTIARCH} include GL OSMesa)
 ELSE (WIN32)
     SET_UP_THIRD_PARTY(MESA lib include MesaGL OSMesa)
 
